corehq/apps/hqmedia/static/hqmedia/js/yui_config.js

```diff
@@ -0,0 +1,9 @@
+//not migrated now, else YUI wont be accessible as local variable for non migrated multimedia pages.
```

Comment:
I used hqDefine(unmigrated) because it follows the existing standard of hq.
please let me know if following below pattern
```
(function(root, factory){
if(typeof define == function){
define('module name', [...], factory)
}else{
factory(YUI)
}
}
)(this, function(){
})
```
makes more sense.

Reply:
`hqDefine` is fine. To respond to the code comment, the way to do this to handle both migrated and unmigrated pages is to do two things:
1) Add `yui-uploader` as a dependency to this module. Since you have it configured in `requirejs_config.js` to export `YUI`, this will give you access to `YUI` on migrated pages.
2) Add a line to the `thirdPartyGlobals` object in `hqDefine` that maps `yui-loader` to `YUI`. Then on unmigrated pages, when `hqDefine` sees the `yui-loader` dependency, it'll grab the global `YUI` and pass it in.

Reply:
I considered That solution. And it worked also. But What stopped me doing that is the notion that `thirdPartyGlobals` should only contain the actually globals which are used globally like jquery etc

Reply:
Nah, `thirdPartyGlobals` isn't just be the major libraries - "globals" is meant to mean globally accessible code, not globally used code. Feel free to rename it if something else would be clearer.

It's definitely preferable to update `thirdPartyGlobals` rather than have this file be a place that we need to remember to come back and update post-migration.